File: tundra-cli/benchmarks/recursive-fibonacci/fib.java
public class fib {
  public static long rec(long n) {
        if (n < 2) return n;
        return rec(n - 1) + rec(n - 2);
    }

    public static void main(String[] args) {
        long n =30;
        System.out.println(rec(n));
    }
}

File: tundra-cli/benchmarks/recursive-fibonacci/fib.cs
// Recursive Fibonacci in C# (Mono)

using System;

public class FibRecursive {
    public static long Fib(long n) {
        if (n < 2) return n;
        return Fib(n - 1) + Fib(n - 2);
    }
    public static void Main(string[] args) {
        long n = 30;
        Console.WriteLine(Fib(n));
    }
}
